### DISCUSSION TITLE
Disconnessioni wifi Shelly 1

### DISCUSSION THREAD

**Discussion by giovannicaliendo** [indomus/forum#1196] [Hardware & Firmware]
Buongiorno a tutti,
non so se il problema è già stato affrontato sul forun, ma ho un certo numero di Shelly 1, sparsi per la casa e connessi ad HA via MQTT , alcuni dei quali periodicamente si disconnettono dal wifi e devo levare la corrente per resettarli. La rete wifi è erogata da un FritzBox 4040 con l'ultimo firmware ed anche gli Shelly sono tutti aggiornati all'ultimo firmware. Le ho provate tutte, ma il problema persiste....Capita anche a voi ?

**Comment by snakuzzo** [comment]
No...tutto regolare. L'unico problema che avevo era in un angolo di una stanza dove ero poco coperto dal WiFi. Problema risolto con un extender
